src/ML-METATOMIC/pair_metatomic.cpp

```diff
@@ -180,19 +185,111 @@ void PairMetatomic::settings(int argc, char ** argv) {
             if (mta_data->uncertainty_threshold <= 0) {
                 error->all(FLERR, "'uncertainty_threshold' in pair_style metatomic must be positive");
             }
-
+            i += 1;
+        } else if (strcmp(argv[i], "variant") == 0) {
+            if (i == argc - 1) {
+                error->all(FLERR, "expected name of off after 'variant' in pair_style metatomic, got nothing");
```

Comment:
```suggestion
                error->all(FLERR, "expected a name or 'off' after 'variant' in pair_style metatomic, got nothing");
```

and same for all the errors below